website/discovery/views.py

```diff
@@ -72,3 +72,13 @@ def activity():
         'popular_public_registrations': popular_public_registrations,
         'hits': hits,
     }
+
+
+def popular_activity_json():
```

Comment:
Since this isn't a view, this doesn't really belong in this file. It probably makes the most sense to just put this function in the script.